### ISSUE TITLE
System.Text.Json IgnoreNullValues not shown

### ISSUE THREAD

**Issue by JulienM28** [dotnet/runtime#48700]
![image](https://user-images.githubusercontent.com/23618436/109009340-b6e85a80-76ae-11eb-8cc4-d755d54e7ff7.png)

I'm using Visual Studio 2019 16.8.6 with .Net 5.0.103 SDK

**Comment by eiriktsarpalis** [comment]
This seems to be by design, it's been marked `EditorBrowsable.Never` due to obsoletion:

https://github.com/dotnet/runtime/commit/c2291eb865fa542c888cefdb16e669dceffeda5f#diff-7ecaf8942a3199fbcbbae5778d2304c8af3527fc79ee03730315d38699e88ba8R202

The [documentation for the property](https://docs.microsoft.com/en-us/dotnet/api/system.text.json.jsonserializeroptions.ignorenullvalues?view=net-5.0) doesn't seem to indicate that it has been obsoleted, perhaps it might make sense to update it? cc @layomia

**Comment by layomia** [comment]
Yup this is by design. Transferring to dotnet/docs to include obsoletion info to the property's docs, after which we can close this issue. cc @tdykstra
